ingest/ingest-datasets.py

```diff
@@ -82,7 +82,7 @@ def create_document( self, entity ):
         if abstract:
             doc.update({"abstract": abstract})
 
-        publication_year = list(ds.objects(DCO.yearOfPublication))
+        publication_year = list(ds.objects(DCT.issued))
```

Comment:
@mrpatrickwest had suggested grabbing the year client-side, otherwise I suppose we lose the month and day portions, which are all "01/01" so far.